dist/profile/manifests/l10n_server.pp

```diff
@@ -0,0 +1,44 @@
+#
+# Accept submissions from the translation plugin
+# containerized in https://github.com/jenkins-infra/l10n-server
+class profile::l10n_server (
+  # Parameters supplied by Hiera
+  $image_tag = 'latest',
+) {
+  include profile::docker
+
+  validate_string($image_tag)
+  $user = 'l10n'
+  $image = 'jenkinsciinfra/l10n-server'
+
+  docker::image { $image:
+    image_tag => $image_tag,
+  }
+
+  docker::run { 'l10n':
+    volumes  => ['/home/l10n/data:/var/l10n'
+    ],
+    username => 'l10n',
+    image    => "${image}}:${image_tag}",
+    require  => [Docker::Image[$image],
+    ],
+  }
+
+  # The File[/etc/init/docker-ircbot.conf] resource is declared by the
+  # module, but we still need to punt the container if the config changes
+  File <| title == '/etc/init/docker-l10n.conf' |> {
+    notify  => Service['docker-l10n'],
+  }
+
+  user { $user:
+    shell      => '/bin/false',
+    managehome => true,
+  }
+
+  file { '/home/l10n/data':
```

Comment:
Since this is not a "real" user (like `kohsuke` or `rtyler`) could we move the home directory to somewhere else like `/srv/l10n` or `/var/lib/l10n`, that would make it more consistent with our other services like mirrorbrain or jenkins